pywb/apps/frontendapp.py

```diff
@@ -116,19 +119,32 @@ def _init_routes(self):
             coll_prefix = '/<coll>'
             self.url_map.add(Rule('/', endpoint=self.serve_home))
 
-        self.url_map.add(Rule(coll_prefix + self.cdx_api_endpoint, endpoint=self.serve_cdx))
-        self.url_map.add(Rule(coll_prefix + '/', endpoint=self.serve_coll_page))
-        self.url_map.add(Rule(coll_prefix + '/timemap/<timemap_output>/<path:url>', endpoint=self.serve_content))
-
-        if self.recorder_path:
-            self.url_map.add(Rule(coll_prefix + self.RECORD_ROUTE + '/<path:url>', endpoint=self.serve_record))
+        self._init_coll_routes(coll_prefix)
 
         if self.proxy_prefix is not None:
             # Add the proxy-fetch endpoint to enable PreservationWorker to make CORS fetches worry free in proxy mode
             self.url_map.add(Rule('/proxy-fetch/<path:url>', endpoint=self.proxy_fetch,
                                   methods=['GET', 'HEAD', 'OPTIONS']))
         self.url_map.add(Rule(coll_prefix + '/<path:url>', endpoint=self.serve_content))
 
+    def _init_coll_routes(self, coll_prefix):
```

Comment:
Documentation (new method)
 - description
 - params
 - param types

New method documentation requirements applies to all new methods added by this PR